.circleci/build.sh

```diff
@@ -127,9 +127,25 @@ echo "  > Installing other dev dependencies."
 php -d memory_limit=-1 "$(command -v composer)" --working-dir="${BUILD_DIR}" require --dev \
   dealerdirect/phpcodesniffer-composer-installer \
   phpspec/prophecy-phpunit:^2 \
-  mglaman/drupal-check \
-  palantirnet/drupal-rector
-cp "${BUILD_DIR}/vendor/palantirnet/drupal-rector/rector.php" "${BUILD_DIR}/."
+  phpstan/extension-installer \
+  mglaman/phpstan-drupal:^1.2 \
+  palantirnet/drupal-rector:^0.18 \
+  friendsoftwig/twigcs:^6.2
+
+# PHPCS config.
```

Comment:
please bring them together (no comments or new lines) and add 
```
echo "  > Copying tools configuration files."
```

Reply:
Updated it.